apps/meteor/server/startup/migrations/v289.ts

```diff
@@ -0,0 +1,87 @@
+import { LivechatRooms, Messages, OmnichannelServiceLevelAgreements } from '@rocket.chat/models';
+import type { IRocketChatRecord } from '@rocket.chat/core-typings';
+import { LivechatPriorityWeight } from '@rocket.chat/core-typings';
+import type { Db } from 'mongodb';
+
+import { addMigration } from '../../lib/migrations';
+import { isEnterprise } from '../../../ee/app/license/server';
+import { BaseRaw } from '../../models/raw/BaseRaw';
+import { db } from '../../database/utils';
+
+interface IOldLivechatPriority extends IRocketChatRecord {
+	_id: string;
+	name: string;
+	description: string;
+	dueTimeInMinutes: number;
+}
+
+class OldLivechatPriorityRaw extends BaseRaw<IOldLivechatPriority> {
```

Comment:
you don't need to create a model for a non-existent collection.. instead you can use the MongoDB driver directly by doing, for example:

```typescript
const { mongo } = MongoInternals.defaultRemoteCollectionDriver();
const messages = mongo.db.collection('rocketchat_livechat_priority');